src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue17453.cs

```diff
@@ -64,6 +64,33 @@ public void EntryClearButtonWorks()
 
 			ClassicAssert.IsEmpty(rtlEntryText);
 		}
+
+		[Test]
+		[Category(UITestCategories.Entry)]
+		public void EntryWithMarginClearButtonWorks()
+		{
+			// https://github.com/dotnet/maui/issues/25225
+
+			App.WaitForElement("WaitForStubControl");
+
+			string? entryText = App.FindElement("EntryWithMargin").GetText();
+
+			if (String.IsNullOrWhiteSpace(entryText))
+				App.EnterText("EntryWithMargin", "Simple Text");
+
+			var entryRect = App.FindElement("EntryWithMargin").GetRect();
+
+			// Set focus
+			App.TapCoordinates(entryRect.X, entryRect.Y);
+
+			// Tap Clear Button
+			var margin = 30;
+			App.TapCoordinates(entryRect.X + margin, entryRect.Y + margin);
+
+			entryText = App.FindElement("EntryWithMargin").GetText();
+
+			ClassicAssert.IsEmpty(entryText);
```

Comment:
The test is failing on Android API 30
![image](https://github.com/user-attachments/assets/f21edb75-f66c-447a-b58f-aa1c54be3f43)

Reply:
@jsuarezruiz can you please /azp?